app/controllers/admin/turn_off_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Admin
+  class TurnOffController < ApplicationController
+    before_action :login_required
+
+    def show
```

Comment:
I gotta ask...  Why two controllers for this?  Why not a single controller that takes a parameter?  I mean, sure, they're tiny, so who cares!  I was just wondering what your thought process was?

Reply:
I thiiiink i was just trying to preserve a route name that made sense, and i couldn't think of a general word for "turn" that would work for both "on" and "off"~! But if you can...

Reply:
Yeah, "turn" is a bit inscrutable isn't it! :)  But maybe it could just be a GET to /admin?  No worries.

Out of curiosity, if we need to tell Rails to pluralize "account" as "account", why don't we need to do the same with "admin"?

Reply:
re: `Admins` — No idea!
re: turn_on/turn_off = get to `AdminController` with params, hmmm. Yes.... Lemme try that.